website/source/guides/web-ui/operating-a-job.html.md

```diff
@@ -0,0 +1,218 @@
+---
+
+# Reviewing Job and Allocation Status
+
+The Web UI can be a powerful companion when monitoring and debugging jobs running in Nomad. The Web
+UI will list all jobs, link jobs to allocations, allocations to client nodes, client nodes to driver
+health, and much more. This creates a fluid exploratory experience.
+
+## Reviewing All Jobs
+
+The first page you will see in the Web UI is the Jobs List page. Here you will find every job for a
+namespace in a region. The table of jobs is searchable, sortable, and filterable. Each job row in
+the table shows basic information, such as job name, status, type, and priority, as well as richer
+information such as a visual representation of all allocation statuses.
+
+This view will also live-update as jobs get submitted, get purged, and change status.
+
+[![Jobs List][img-jobs-list]][img-jobs-list]
+
+## Filtering Jobs
+
+If your Nomad cluster has many jobs, it can be useful to filter the list of all jobs down to only
+those matching certain facets. The Web UI has four facets you can filter by:
+
+1. **Type:** The type of job, including Batch, Parameterized, Periodic, Service, and System.
+2. **Status:** The status of the job, including Pending, Running, and Dead.
+3. **Datacenter:** The datacenter the job is running in, including a dynamically generated list
+   based on the jobs in the namespace.
+4. **Prefix:** The possible common naming prefix for a job, including a dynamically generated list
+   based on job names up to the first occurrence of `-`, `.`, and `_`. Only prefixes that match
+   multiple jobs are included.
+
+[![Job Filters][img-job-filters]][img-job-filters]
+
+## Monitoring an Allocation
+
+In Nomad, allocations are the schedulable units of work. This is where runtime metrics begin to
+surface. An allocation is composed of one or more tasks, and the utilization metrics for tasks are
+aggregated so they can be observed at the allocation level.
+
+### Resource Utilization
+
+Nomad has APIs for reading point-in-time resource utilization metrics for tasks and allocations. The
+Web UI uses these metrics to create time-series graphs for the current session.
+
+When viewing an allocation, resource utilization will automatically start logging.
+
+[![Allocation Resource Utilization][img-alloc-resource-utilization]][img-alloc-resource-utilization]
+
+### Task Events
+
+When Nomad places, prepares, and starts a task, a series of task events are emitted to help debug
+issues in the event that the task fails to start.
+
+Task events are listed on the Task Detail page and live-update as Nomad handles managing the task.
+
+[![Task Events][img-task-events]][img-task-events]
+
+### Rescheduled Allocations
+
+Allocations will be placed on any client node that satisfies the constraints of the job definition.
+However, just because a node sounds like a good fit doesn't mean there isn't the possibility of
+unforeseen hostility, (e.g., corrupted `/bin`, no access to a container registry).
```

Comment:
I would change this line and take out the part about **unforeseen hostility** and just make it something like this:

**There are events, however, that will cause Nomad to reschedule allocations (ex: node failures).**

Reply:
Similar to the above, I was trying to provide more specific examples, since "node failures" is still broad. 

Do you think the specificity is distracting?

Reply:
After reading through this again, I actually appreciate the context and specificity. To keep it a bit more concise, maybe we could take those extra failure reasons and tack it on to the sentence I suggested:

**There are events, however, that will cause Nomad to reschedule allocations (ex: node failures, corrupted `/bin`, losing access to container registry).*